extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -21,21 +21,31 @@ import {removeFragment} from '../../../src/url';
 
 class AmpAccordion extends AMP.BaseElement {
 
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {?NodeList} */
+    this.sections_ = null;
+
+    /** @private {?string} */
+    this.id_ = null;
+
+    /** @private {?Object|undefined} */
```

Comment:
Can this be changed to either not accept null or undefined? If no, please document what they mean.